benchmarks/common.py

```diff
@@ -0,0 +1,21 @@
+import random
+from typing import List
+
+from traveltimepy import Location, Coordinates
```

Comment:
Do we need the "py" ending in the package name? Seems redundant. Google has simple "import googlemaps", not "googlemapspy".

Reply:
yeah, I also wanted to use traveltime but it was rejected

Reply:
so I renamed it back

Reply:
Rejected by a reviewer? Well it's a change of API, but since 3.0 is not backward compatible I think it's ok

Reply:
Yeah, I was asked to rename it back